saleor/graphql/order/tests/test_order.py

```diff
@@ -6026,6 +6026,57 @@ def test_order_query_with_filter_status(
     assert order_id in orders_ids_from_response
 
 
+def test_order_query_with_ready_to_fulfill_filter_status(
```

Comment:
Do you think that this is good idea to extend  benchmark tests also?
`saleor.graphql.order.tests.benchmark.test_order`

Reply:
They seem to be checking the numer of queries and so far it's constant here